gui/healthbar.py

```diff
@@ -0,0 +1,19 @@
+import pygame
+
+class HealthBar():
```

Comment:
It's better to make it as a universal class for all kind of points (mana, stamina)